static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-codes.js

```diff
@@ -69,7 +69,7 @@ const getCode = function(data: {
     slot: string,
     breakpoint: string,
     section: string,
-}): { code: string } {
+}): { code?: string, image?: string, text?: string } {
```

Comment:
Yeah I had looked at something similar to this, but using a 'union' type means having to do a load of type-checking anywhere I'm using the values. Seemed to not be worth it, though it does read slightly better here.